sycl/include/sycl/handler.hpp

```diff
@@ -1633,7 +1633,7 @@ class __SYCL_EXPORT handler {
   void setStateSpecConstSet();
   bool isStateExplicitKernelBundle() const;
 
-  std::shared_ptr<detail::kernel_bundle_impl>
+  std::shared_ptr<detail::kernel_bundle_impl> &
```

Comment:
I'm not sure is it API change or not. `getOrInsertHandlerKernelBundle()` is not supposed to be used externally.